### PR TITLE
CHE-1787: Remove workspace id from websocket connection path to master

### PR DESCRIPTION
**Pull request by vinokurig**
### What does this PR do?

Removes workspace id from websocket connection path to master.
### What issues does this PR fix or reference?
#1787
### Previous behavior

Separate web-socket connection was created for each workspase.
### New behavior

All workspaces are using one web-socket connection.

@vparfonov @akurinnoy @ashumilova Please review

### CODE REVIEW

File: dashboard/src/components/api/che-websocket.factory.ts [hunk 1]
```diff
@@ -54,17 +54,12 @@ export class CheWebsocket {
   }
 
 
-  getBus(workspaceId) {
-    var currentBus = this.sockets.get(workspaceId);
-    if (!currentBus) {
+  getBus() {
```
Comment (benoitf) [inline dashboard/src/components/api/che-websocket.factory.ts:58]: this kind of change needs to be applied on getRemoteBus as well

**Review by benoitf — CHANGES_REQUESTED** [review]

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -23,7 +23,7 @@
     </context-param>
     <context-param>
         <param-name>org.eclipse.che.websocket.endpoint</param-name>
-        <param-value>/ws/{ws-id}</param-value>
+        <param-value>/ws</param-value>
```
Comment (benoitf) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/web.xml:26]: could we have http redirect from /ws/<id> to /ws ?
for backward compliance for existing clients ?

Comment (vinokurig) [reply assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/web.xml:26]: This is a network issue. This needs investigation.

Comment (benoitf) [reply assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/web.xml:26]: ok

**Review by benoitf — APPROVED** [review]
Except the fact that I would want to have a redirect from /ws/<id> to /ws for any existing clients, ok for me

**Review by akurinnoy — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Review by ashumilova — APPROVED** [review]

**Comment by benoitf** [comment]
@vinokurig  is that https://github.com/eclipse/che/blob/47193dc85bcfd00f2dc03dc7f86c22f4cf6dd4e4/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceServiceLinksInjector.java#L133-L135 should have been updated ?

**Comment by vinokurig** [comment]
@benoitf Thank you, I will fix that.

**Comment by benoitf** [comment]
@vinokurig I've found another location, I will provide PR to fix it

**Comment by vinokurig** [comment]
@benoitf ok, thanks

**Pull request merged**